src/BenchmarkDotNet/Helpers/SourceCodeHelper.cs

```diff
@@ -42,7 +42,7 @@ public static string ToSourceCode(object value)
 
             switch (value) {
                 case TimeInterval interval:
-                    return "new BenchmarkDotNet.Horology.TimeInterval(" + ToSourceCode(interval.Nanoseconds) + ")";
+                    return "new BenchmarkDotNet.Horology.TimeValue(" + ToSourceCode(interval.Nanoseconds) + ")";
```

Comment:
this should make the CI green

```suggestion
                    return "new BenchmarkDotNet.Horology.TimeInterval(" + ToSourceCode(interval.Nanoseconds) + ")";
```